app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,39 @@
 <html>
   <head>
     <title>BillingPaymentSystem</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name='viewport' content='width=device-width,initial-scale=1'>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
-  </body>
+     <% if user_signed_in? && current_user.role == 'admin' %>
+       <%= render partial: 'shared/navbar_admin' %>
+     <% elsif user_signed_in? %>
+       <%= render partial: 'shared/navbar_buyer' %>  
+     <% end %>
+     <div class='container'>
+       <%= render partial: 'shared/flash' %>
+
+       <% if user_signed_in? %>
+         <p class="my-4">Hi <strong><%= current_user.name %></strong> </p>
+         <p>your role is <%= current_user.role %></p>
+           Logged in as: <strong><%= current_user.email %> </strong>
+           <hr>
```

Comment:
remove following

<p class="my-4">Hi <strong><%= current_user.name %></strong> </p>
         <p>your role is <%= current_user.role %></p>
           Logged in as: <strong><%= current_user.email %> </strong>
           <hr>